app/views/attachments/upload.js.erb

```diff
@@ -1,51 +1,55 @@
-<%-
-  _all_categories = AttachmentCategory.all.collect { |p| [p.name, p.id]} 
-  _disable_auto_completes = Setting['plugin_redmine_attachment_categories']['disable_auto_completes'] == "true"
-  
-  _attachment_tag  = @attachment.thumbnailable? ? thumbnail_tag( @attachment ) : attachment_category_tag( @attachment.attachment_category, :span )
-  _category_select = attachment_category_select(
-    "attachments[#{j params[:attachment_id]}][attachment_category_id]",
-    _all_categories, @attachment.attachment_category_id,
-    :onchange => "changeCategoryColor(this, '#attachments_#{params[:attachment_id]}')"
-  )
-  
-  _file_or_thumbnail = j content_tag( 
-    :div, 
-    _attachment_tag + tag(:br) + _category_select,
-    {:class => "attachment_categories_column",
-     :style => "width:#{Setting.thumbnails_size.to_i}px;float:left;word-break:break-all;"
-    }
-  )
-  
-%>
 var fileSpan = $('#attachments_<%= j params[:attachment_id] %>');
-
 <% if @attachment.new_record? %>
   fileSpan.hide();
   alert("<%= escape_javascript @attachment.errors.full_messages.join(', ') %>");
-  
 <% else %>
+  <% if false %>
```

Comment:
I want to use `attachment.container_type != 'Issue'` or `controller.controller_name != 'issue'`, but `attachment.container_type` is empty during creating attachment, and `controller.controller_name` is always `attachment`, so this may be harder than I expected.

Reply:
About above issue, I solved it by referring the following article.
* [Railsで遷移元のcontroller、action名などを取得する方法とエラー対応 #Rails - Qiita](https://qiita.com/hrt_dev/items/3209a5debfd2f5760418)